consensus/safety-rules/src/safety_rules.rs

```diff
@@ -312,8 +312,6 @@ impl SafetyRules {
         self.verify_last_vote_round(proposed_block.block_data())?;
 
         let vote_data = self.extension_check(vote_proposal)?;
-        self.persistent_storage
-            .set_last_voted_round(proposed_block.round())?;
```

Comment:
wait if I understand this code correctly you need to keep this, you can remove the previous one though

Reply:
oh no nevermind, I wasn't looking down. Why the one on line 308 though?